boilerplates/lucia-auth/files/$package.json.ts

```diff
@@ -0,0 +1,17 @@
+import { addDependency, loadAsJson, type TransformerProps } from "@batijs/core";
+
+export default async function getPackageJson(props: TransformerProps) {
+  const packageJson = await loadAsJson(props);
+
+  return addDependency(packageJson, await import("../package.json").then((x) => x.default), {
+    devDependencies: ["dotenv", "@types/better-sqlite3", "@types/cookie"],
```

Comment:
`@types/better-sqlite3` is still needed even without drizzle, since we’re using `better-sqlite3` for the database. 
```
Could not find a declaration file for module 'better-sqlite3'
```